src/main/java/org/spongepowered/api/stats/StatisticBuilder.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * This file is part of Sponge, licensed under the MIT License (MIT).
+ *
+ * Copyright (c) SpongePowered.org <http://www.spongepowered.org>
+ * Copyright (c) contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+package org.spongepowered.api.stats;
+
+import org.spongepowered.api.text.translation.Translation;
+
+/**
+ * Represents a builder interface to create new instances of {@link Statistic}s.
+ */
+public interface StatisticBuilder {
+
+    /**
+     * Sets the translation for the {@link Statistic}.
+     *
+     * @param translation The translation for the statistic
+     * @return This builder
+     */
+    StatisticBuilder translation(Translation translation);
+
+    /**
+     * Sets the id used to save the current values of the {@link Statistic}.
+     *
+     * @param id The id used to save the current values of the statistic
+     * @return This builder
+     */
+    StatisticBuilder id(String id);
+
+    /**
+     * Sets the unit the {@link Statistic} will be measured in.
+     *
+     * @param unit The unit the statistic will be measured in
+     * @return This builder
+     */
+    StatisticBuilder unit(StatisticUnit unit);
+
+    /**
+     * Sets the {@link StatisticType} the {@link Statistic} belongs
+     * to.
+     *
+     * @param type The statistic type the grouped statistic belongs to
```

Comment:
Should be `@param type The statistic type the statistic belongs to`
(without group)
Because everything has a type now.